admin/database/postgres/postgres.go

```diff
@@ -1020,6 +1020,12 @@ func (c *connection) DeleteExpiredUserAuthTokens(ctx context.Context, retention
 	return parseErr("auth token", err)
 }
 
+// DeleteInactiveUserAuthTokens deletes user authentication tokens that have not been used within the specified retention period.
+func (c *connection) DeleteInactiveUserAuthTokens(ctx context.Context, retention time.Duration) error {
+	_, err := c.getDB(ctx).ExecContext(ctx, "DELETE FROM user_auth_tokens WHERE used_on IS NULL OR used_on < now() - $1", retention)
```

Comment:
In theory `used_on` might be null right after creation (although I think currently we always populate it). But best to be safe here